builders/build/eth-api/dev-env/hardhat.md

```diff
@@ -234,13 +280,14 @@ main()
   });
 ```
 
-您现在可以使用`run`命令并指定`moonbase`作为网络来部署`Box.sol`合约：
+使用以下命令来运行脚本，部署`Box.sol`合约；您需要给这个命令定义一个部署合约的网络名称，这个名称已在`hardhat.config.js`中定义。
```

Comment:
In the above code snippet, the comments were changed from `// 2. Instantiating a new Box smart contract` to `// 2. Instantiate a new Box smart contract`

And similarly, `  // 3. Waiting for the deployment to resolve` was changed to `  // 3. Wait for the deployment to resolve`

Some of the comments in the other snippets were translated to CN. I feel like if we are doing some snippets, we should just do all of them so we are being consistent

Reply:
Done!